### PR TITLE
Update Misc Batch

### PR DESCRIPTION
**Pull request by ydFu**
* [zh] Umbrella issue: pages out of sync in concepts section #26177(Misc Batch 4)

```
[x] content/zh/docs/concepts/architecture/controller.md
[x] content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md
[x] content/zh/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md
[x] content/zh/docs/concepts/extend-kubernetes/operator.md
```

* sync with english version in home/_index.md and tool/_index.md

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Comment by tengqm** [comment]
Would you like work on this in batches and link the PRs to issue #26177?

**Comment by ydFu** [comment]
/retitle [zh] Sync concepts pages for architecture and extend-kubernetes

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -348,11 +348,11 @@ gRPC 服务通过 `/var/lib/kubelet/pod-resources/kubelet.sock` 的 UNIX 套接
 如果设备监控代理以 DaemonSet 形式运行，必须要在插件的
 [PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core)
 中声明将 `/var/lib/kubelet/pod-resources` 目录以
-{{< glossary_tooltip text="卷" term_id="volume" >}}的形式被挂载到容器中。
+{{< glossary_tooltip text="卷" term_id="volume" >}}的形式被挂载到设备监控代理中。
 
 对“PodResources 服务”的支持要求启用 `KubeletPodResources`
 [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
-从 Kubernetes 1.15 开始默认启用。
+从 Kubernetes 1.15 开始默认启用，自从Kubernetes 1.20开始为v1。。
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:355]:
```suggestion
从 Kubernetes 1.15 开始默认启用，自从 Kubernetes 1.20 开始为 v1。
```

注意空格。

Comment (ydFu) [reply content/zh/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: thanks.

**Comment by howieyuen** [comment]
Thanks for the help~
/lgtm

**Pull request merged**